packages/@okta/vuepress-theme-prose/assets/css/layouts/_authorizationLayout.scss

```diff
@@ -1,6 +1,6 @@
 .authorization-layout {
   display: flex;
-  background: linear-gradient(180deg, #1D2C33 0%, cv('brand','base') 100%);
+  background: cv('brand','base');
```

Comment:
The gradient runs beyond the bottom of the layout where the divider image is, so it never fully reaches the `brand base` color and causes a small visual break. I decided to just remove the gradient since it's so minor that visually it's imperceptible. Making the bg a solid color is just far simpler and avoids the issues mentioned with blending.